index.js

```diff
@@ -4,15 +4,49 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+updateTitle = header.style.color = "Green"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+let travelerProfile = document.querySelector(".traveler")
+let imgUrl = travelerProfile.querySelector("img")
+imgUrl.src = traveler.photo
+imgUrl.alt = traveler.name
+
+let nameUpdate = travelerProfile.querySelector("h2")
+nameUpdate.textContent = traveler.name
+
+let nicknameUpdate = travelerProfile.querySelector("em")
+nicknameUpdate.textContent = traveler.nickname
 
 
 /***** Deliverable 4 *****/
+animalSightingsArr = traveler.animalSightings
+let animalSightingList = document.querySelector('ul#animals')
+
+animalSightingsArr.forEach(function (sightingObj){
+    let liTag = document.createElement('li')
+    let pTag = document.createElement('p')
+    let imgTag = document.createElement('img')
+    let aTag = document.createElement('a')
+
+    liTag.setAttribute('data-id', sightingObj.id )
```

Comment:
For dataset attributes, you can also use this syntax:

```js
li.dataset.id = sighting.id
```

This will add a `data-id` attribute to the element, like this:

```html
<li data-id="1">...</li>
```